var/spack/repos/builtin/packages/geomodel/package.py

```diff
@@ -18,6 +18,7 @@ class Geomodel(CMakePackage):
 
     license("Apache-2.0", checked_by="wdconinc")
 
+    version("6.7.0", sha256="bfa69062ba191d0844d7099b28c0d6c3c0f87e726dacfaa21dba7a6f593d34bf")
```

Comment:
I cannot confirm this due to `read operation timed out`.

Reply:
Might have been a transient.

```
$ spack checksum --verify --batch geomodel 6.7.0
==> Found 1 version of geomodel
==> Fetching https://gitlab.cern.ch/GeoModelDev/GeoModel/-/archive/6.7.0/GeoModel-6.7.0.tar.bz2
==> Verified 1 of 1
  
  6.7.0  [=] Correct
  
```

Reply:
🤔 I still get read operation timed out. Perhaps it's related to being on VPN?